integration_tests/go.mod

```diff
@@ -117,5 +117,6 @@ require (
 
 replace (
 	github.com/go-ldap/ldap/v3 => github.com/YangKeao/ldap/v3 v3.4.5-0.20230421065457-369a3bab1117
+	github.com/pingcap/tidb v1.1.0-beta.0.20240111112854-1ad36eb0ef29 => github.com/lance6716/tidb v1.1.0-beta.0.20240122070601-bddfc7704131
```

Comment:
How about modifying mock PD client in mockstore to make tidb-compatibility CI happy

Reply:
Do you mean `internal/mockstore/mocktikv/pd.go`? I have updated that file. Maybe "tidb-compatibility" CI will use master branch of TiDB so before TiDB upgrade PD client and client-go it can't build.

Reply:
OK, I forgot it.